arcgis-ios-sdk-samples/Edit data/Edit with branch versioning/EditWithBranchVersioningViewController.swift

```diff
@@ -0,0 +1,542 @@
+        let alertController = UIAlertController(
+            title: "Choose to switch to another version",
+            message: nil,
+            preferredStyle: .actionSheet
+        )
+        existingVersionNames.forEach { name in
+            let action = UIAlertAction(title: name, style: .default) { _ in
+                completion(name)
+            }
+            alertController.addAction(action)
+        }
+        let cancelAction = UIAlertAction(title: "Cancel", style: .cancel)
+        alertController.addAction(cancelAction)
+        alertController.popoverPresentationController?.barButtonItem = sender
+        present(alertController, animated: true)
+    }
+    
+    func chooseVersionAccessPermission(_ sender: UIBarButtonItem, completion: @escaping (AGSVersionAccess) -> Void) {
+        let alertController = UIAlertController(
+            title: "Choose an access level for the new branch version",
+            message: nil,
+            preferredStyle: .actionSheet
+        )
+        let versionAccessPermission: [AGSVersionAccess] = [.public, .protected, .private]
+        versionAccessPermission.forEach { versionAccess in
+            let action = UIAlertAction(title: versionAccess.title, style: .default) { _ in
+                completion(versionAccess.self)
+            }
+            alertController.addAction(action)
+        }
+        let cancelAction = UIAlertAction(title: "Cancel", style: .cancel)
+        alertController.addAction(cancelAction)
+        alertController.popoverPresentationController?.barButtonItem = sender
+        present(alertController, animated: true)
+    }
+    
+    // MARK: UIViewController
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Add the source code button item to the right of navigation bar.
+        (navigationItem.rightBarButtonItem as? SourceCodeBarButtonItem)?.filenames  = ["EditWithBranchVersioningViewController"]
+        // Load the service geodatabase.
+        let damageFeatureService = URL(string: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/DamageAssessment/FeatureServer")!
+        serviceGeodatabase = loadServiceGeodatabase(from: damageFeatureService)
+    }
+}
+
+// MARK: - AGSGeoViewTouchDelegate
+
+extension EditWithBranchVersioningViewController: AGSGeoViewTouchDelegate {
+    func geoView(_ geoView: AGSGeoView, didTapAtScreenPoint screenPoint: CGPoint, mapPoint: AGSPoint) {
+        // Dismiss any presenting callout.
+        mapView.callout.dismiss()
+        // Disable features editing on the default branch.
+        if currentVersionName == defaultVersionName {
+            // Tap to identify a pixel on the feature layer.
+            identifyPixel(on: featureLayer, at: screenPoint) { feature in
+                // Show a callout without the accessory button.
+                self.showCallout(for: feature, tapLocation: mapPoint, isAccessoryButtonHidden: true)
+            }
+            return
```

Comment:
Instead of returning here, could what comes after this `if` statement be moved into an `else` clause?